keras_adversarial/adversarial_model.py

```diff
@@ -74,7 +75,8 @@ def adversarial_compile(self, adversarial_optimizer, player_optimizers, loss, co
         self.optimizer = None
 
         # Build player models
-        for opt, model in zip(self.optimizers, self.layers):
+        for opt, model, compile_kwargs in itertools.zip_longest(
```

Comment:
can just use regular `zip` if the default is the right length. `zip_longest` is failing the py2 tests on travis.